tools/ci_build/build.py

```diff
@@ -184,9 +183,9 @@ def parse_arguments():
         available test data directories.""")
 
     # generate documentation
-    parser.add_argument(
-        "--gen_doc", action='store_true',
-        help="Generate documentation on contrib ops")
+    parser.add_argument("--gen_doc", nargs='?', const='yes', type=str,
+                        help="Generate operator kernels and contrib operator documentation. "
```

Comment:
nit: Generate documentation listing standard ONNX operators and types implemented by various execution providers and contrib operator schemas.